sudsssltransport.py

```diff
@@ -65,10 +65,10 @@ def open(self, request):
             )
             response.raise_for_status()
         except SSLError as e:
-            raise TransportError(e.message, None)
+            raise TransportError(str(e), None)
         except (RequestException, HTTPError, ConnectionError) as e:
             raise TransportError(str(e), None)
-        return StringIO(response.text)
+        return suds.BytesIO(response.content)
```

Comment:
Aaah!  So that's how to return a `StringIO` object that doesn't have "buffer interface" Exceptions.  I was trying to get python 3 working locally, but didn't know about `suds.BytesIO`